test/framework/github.py

```diff
@@ -1317,26 +1318,96 @@ def test_github_create_test_report(self):
                 'log_file': logfile,
             }),
         ]
+        environ = {
+            'USER': 'test',
+            'DONT_REMOVE_ME': 'test-123',
+        }
+        JWT_HDR = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9'
+        JWT_PLD = 'eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNzA4MzQ1MTIzLCJleHAiOjE3MDgzNTUxMjN9'
+        JWT_SIG = 'SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c'
+        secret_environ = {
+            'REMOVEME': 'test-abc',
+            'ReMoVeMe2': 'TeSt-xyz',
+
+            'AWS_ACCESS_KEY': 'AKIAIOSFODNN7EXAMPLE',
+            'AWS_SECRET_KEY': 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY',
+            'JWT': '.'.join([JWT_HDR, JWT_PLD, JWT_SIG]),
+            'GH_TOKEN': 'ghp_123456789_ABCDEFGHIJKlmnopqrstuvwxyz',
```

Comment:
```suggestion
            'GH_TOKEN': 'ghp_123456789_ABCDEFGHIJKlmnopqrstuvwxyz',
            'SAME_THING_DIFFERENT_NAME': 'ghp_123456789_ABCDEFGHIJKlmnopqrstuvwxyz',
```

Reply:
Let's also check the pattern matching on the value is working.

Reply:
This is really because the regex may be incorrect but it would still be excluded because of `TOKEN` in the name.

Reply:
This is fixed, see changes in ec5dc94f124ca3c467eb6bd5a11f6fae267250b1